### PR TITLE
Allow overriding the exclude-limit in DisabledConfigReporter

### PR DESCRIPTION
**Pull request by jdelStrother**
It's currently hard-coded to 15.  WDYT to this change?

### CODE REVIEW

File: lib/haml_lint/reporter/disabled_config_reporter.rb [hunk 1]
```diff
@@ -23,10 +23,12 @@ def self.available?
     # Create the reporter that will display the report and write the config.
     #
     # @param _log [HamlLint::Logger]
-    def initialize(_log)
-      super
+    def initialize(_log, limit: 15)
+      super(_log)
       @linters_with_lints = Hash.new { |hash, key| hash[key] = [] }
       @linters_lint_count = Hash.new(0)
+      puts "New limit: #{limit}"
```
Comment (sds) [inline lib/haml_lint/reporter/disabled_config_reporter.rb]: Did you mean to leave this `puts` in here?

Comment (jdelStrother) [reply lib/haml_lint/reporter/disabled_config_reporter.rb]: Oops, no. Will fix it

**Comment by jdelStrother** [comment]
Done, let me know what you think.

**Comment by sds** [comment]
Can you fix the [RuboCop errors](https://travis-ci.org/brigade/haml-lint/jobs/517245235)? The Metrics/LineLength ones can be fixed by appending 

```ruby
# rubocop:disable Metrics/LineLength
```

...to the end of the line in question. In this case, I don't think the lint is providing value, so let's silence.

**Comment by jdelStrother** [comment]
Sorry about the delay, I've fixed up the linting errors.

**Pull request merged**